.github/CODEOWNERS

```
@@ -0,0 +1,238 @@
+/helmfile.d/upstream/kubernetes-metrics-server/ @elastisys/goto-monitoring-stack
+/helmfile.d/upstream/minio/ @elastisys/goto-block-storage
+/helmfile.d/upstream/open-policy-agent-gatekeeper/ @elastisys/goto-security
+/helmfile.d/upstream/opensearch-project/ @elastisys/goto-logging-stack
+/helmfile.d/upstream/projectcalico/ @elastisys/goto-kubernetes
+/helmfile.d/upstream/prometheus-community/kube-prometheus-stack/ @elastisys/goto-monitoring-stack
+/helmfile.d/upstream/prometheus-community/prometheus-blackbox-exporter/ @elastisys/goto-monitoring-stack
+/helmfile.d/upstream/prometheus-community/prometheus-elasticsearch-exporter/ @elastisys/goto-logging-stack
+/helmfile.d/upstream/vmware-tanzu/velero/ @elastisys/goto-security
+
+
+#
+# helmfile.d/values
+#
+
+/helmfile.d/values/external-dns/ @elastisys/goto-ingress-auth
+/helmfile.d/values/falco/ @elastisys/goto-security
+/helmfile.d/values/fluentd/ @elastisys/goto-logging-stack
+/helmfile.d/values/gatekeeper/ @elastisys/goto-security
+/helmfile.d/values/grafana/ @elastisys/goto-monitoring-stack
+/helmfile.d/values/harbor/ @elastisys/goto-container-registry
+/helmfile.d/values/hnc/ @elastisys/goto-security
+/helmfile.d/values/networkpolicies/ @elastisys/goto-security
+/helmfile.d/values/networkpolicy/ @elastisys/goto-security
+/helmfile.d/values/opensearch/ @elastisys/goto-logging-stack
+/helmfile.d/values/podsecuritypolicies/ @elastisys/goto-security
+/helmfile.d/values/rclone/ @elastisys/goto-security
+/helmfile.d/values/thanos/ @elastisys/goto-monitoring-stack
+/helmfile.d/values/trivy/ @elastisys/goto-security
+/helmfile.d/values/userCRDs/ @elastisys/goto-security
+/helmfile.d/values/autoscaling-monitoring.yaml.gotmpl @elastisys/goto-kubernetes
+/helmfile.d/values/calico-accountant.yaml.gotmpl @elastisys/goto-kubernetes
+/helmfile.d/values/calico-default-deny.yaml.gotmpl @elastisys/goto-kubernetes
+/helmfile.d/values/cert-manager.yaml.gotmpl @elastisys/goto-ingress-auth
+/helmfile.d/values/cluster-admin-rbac.yaml.gotmpl @elastisys/goto-security
+/helmfile.d/values/dev-namespaces.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/dex.yaml.gotmpl @elastisys/goto-ingress-auth
+/helmfile.d/values/file-copier.yaml.gotmpl @elastisys/goto-ingress-auth
+/helmfile.d/values/ingress-nginx-probe-ingress.yaml.gotmpl @elastisys/goto-ingress-auth
+/helmfile.d/values/ingress-nginx.yaml.gotmpl @elastisys/goto-ingress-auth
+/helmfile.d/values/kubeapi-metrics.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/kube-prometheus-stack-* @elastisys/goto-monitoring-stack
+/helmfile.d/values/kube-state-metrics-extra-resource-metrics.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/kured.yaml.gotmpl @elastisys/goto-security
+/helmfile.d/values/letsencrypt.yaml.gotmpl @elastisys/goto-ingress-auth
+/helmfile.d/values/metrics-server.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/minio.yaml.gotmpl @elastisys/goto-block-storage
```

Should this be:
```suggestion
/helmfile.d/values/minio.yaml.gotmpl @elastisys/goto-appdev-experience 
```
Since it's only used for local dev purpose

---

I think that app-dev is not the correct goto area, since that is focused on app devs that use this platform not the contributors that develop this platform.
So if we want to move it, then QA might be more appropriate.

---

Yea, I think that makes more sense than block-storage at least since this is in fact object storage.

---

Ok, I'll change it.